keps/sig-api-machinery/1693-warnings/README.md

```diff
@@ -79,27 +79,27 @@ and manually sweeping for use of deprecated features.
 * Allow a cluster administrator to programatically identify particular clients using deprecated APIs
 
 ### Non-Goals
-
+ 
 While the proposed warning mechanism is generic enough to carry arbitrary warnings,
 the following items are out of scope for the first iteration of this feature:
 
-* Allowing extensions mechanisms like admission webhooks to contribute warnings
+* Allowing extension mechanisms like admission webhooks to contribute warnings
```

Comment:
This was the original motivation for the first version of this feature introduced by brendan burns.  While I understand a desire to narrow scope, it seems appropriate to consider how APIs without the ability to add apiserver code.  Before this feature goes GA, it seems reasonable to resolve this problem.

Reply:
That's a second design effort that includes the API between the admission webhooks and the API server. I'm in favor of it eventually, but not in this first iteration.

Reply:
> That's a second design effort that includes the API between the admission webhooks and the API server. I'm in favor of it eventually, but not in this first iteration.

It doesn't need to be in this iteration, but I would like to see it on the criteria for graduation to GA. From a plumbing perspective and a client experience perspective, this enhancement is about warnings. Deprecation happens to be the first one we're doing, but you're creating WarningRecorder, using a warning standard, and honoring warnings in the client.

Reply:
added description of CRD and admissionwebhook integrations, and added to GA criteria